### PR TITLE
Better documentation

### PR DESCRIPTION
**Pull request by Devsome**
Regarding to #8

### CODE REVIEW

**Review by whatsinmyopsec — APPROVED** [review]
cool nice one appreciated

File: README.md [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
-## running
----
+3. Copy from your Bot that Token and paste it into the "**Password**" variable.
 
+### Example:
+```md  
+USERNAME="Botname"  
+PASSWORD="NDgyMjM0ODU1MzQygG1xAM1Mzcw.W37ENQ.SxEyK0451hK-lMaYDT-wEHdc"  
+CHANNEL0="Welcome"  
+CHANNEL1="offtopic"
```
Comment (bopojoe) [inline README.md:24]: this should be 

```suggestion
CHANNEL0="Twitch channel name 1"  
CHANNEL1="Twitch channel name 2"
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
+```  
+  
+4. Make sure to install **npm** with these Commands:
```
Comment (bopojoe) [inline README.md:27]:
```suggestion
4. Make sure to install **npm dependancies** with these Commands:
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,34 @@
...
-CHANNEL0=channel you want
-CHANNEL1=second channel you want
-```
+1. Run this command `cp .env.example .env` or by simple rename that `.env.example` to `.env`
+  
+  2. Go to [Discord Developers](https://discord.com/developers/applications) and open / create your Bot Application.
 
-## running
----
+3. Copy from your Bot that Token and paste it into the "**Password**" variable.
```
Comment (bopojoe) [inline README.md:17]:
```suggestion
3. Copy your Bot OAuth Token from twitch tmi [website](https://twitchapps.com/tmi/) and paste it into the "**Password**" variable.
```

**Review by bopojoe — CHANGES_REQUESTED** [review]
this is a better fix and the English is slightly better

**Review by whatsinmyopsec — COMMENTED** [review]
...

**Review by whatsinmyopsec — COMMENTED** [review]
yup

**Pull request merged**